src/App.vue

```diff
@@ -21,18 +21,20 @@
 import RoomConnect from './components/RoomConnect.vue';
 import Chat from './components/Chat.vue';
 import { useVideoStore } from './stores/videoStore';
-// import { useSocketStore } from "./stores/socketStore";
-import { onMounted, Ref, ref } from 'vue';
+import { useSocketStore } from "./stores/socketStore";
+import { inject, onMounted, Ref, ref,  } from 'vue';
 
 const videoStore = useVideoStore();
 
 onMounted(() => {
   videoStore.videoHandler.addVideoEventListeners();
 })
 
- // const socketStore = useSocketStore();  //not used for testing, need to add v-if with isConnected method to Chat
+const socketStore = useSocketStore(); // not used for testing, need to add v-if with isConnected method to Chat
 // and RoomConnect component
 
+socketStore.socket.streamingPlatform = inject('streamingPlatform');
```

Comment:
I haven't seen usage of `provide/inject` this way, it's nice and simple, but I would inject this value in the `createRooom` and `joinRoom` functions in the `RoomConnect.vue` component